nix/tests/default.nix

```diff
@@ -131,4 +131,9 @@ in {
     user = "deploy";
     deployArgs = "-s .#profile -- --offline";
   };
+  hyphen-ssh-opts-regression = mkTest {
+    name = "profile";
+    user = "deploy";
+    deployArgs = "-s .#profile --ssh-opts '-p 22' -- --offline";
```

Comment:
Oh, sorry, `sshOpts` from flake aren't merged with `--ssh-opts`, thus we need to explicitly add `o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null` too for test to pass 🤦 
```suggestion
    deployArgs = "-s .#profile --ssh-opts '-p 22 -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null' -- --offline";
```